multimedia/fluidsynth/Portfile

```diff
@@ -36,27 +35,22 @@ depends_lib         port:flac \
                     port:portaudio \
                     port:readline
 
-patchfiles          yosemite-libtool.patch
-
 configure.args-append \
-                    --disable-dependency-tracking \
-                    --disable-pulse-support \
-                    --disable-jack-support \
-                    --disable-dbus-support
+                    -Denable-jack=OFF \
+                    -Denable-dbus=OFF \
+                    -Denable-pulseaudio=OFF
```

Comment:
I'd append

```
                    -Denable-framework=OFF \
                    -DLIB_SUFFIX="" \
                    -DCMAKE_DISABLE_FIND_PACKAGE_Doxygen=ON
```

This disables the frameworks build (not appropriate for MacPorts IMHO), makes libraries go into `$prefix/lib` (and not $prefix/lib64) and disables the doxygen lookup.

Reply:
OK.